src/App.tsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 
+import { PostList } from './components/PostList/PostList';
+
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts: Posts[] = posts.map(post => ({
```

Comment:
Posts[]
Posts - массив постов
[] - массив

Массив массивов?
```suggestion
const preparedPosts: Post[] = posts.map(post => ({
```